test/isValidLCDate.test.js

```diff
@@ -0,0 +1,52 @@
+const isValidLCDate = require('../utils/isValidLCDate');
+
+let mockDate;
+let mockHour;
+
+test('Valid LC Date', () => {
+  mockDate = '20210121';
```

Comment:
name `mockDate` and `mockHour` explicitly following the parameter names in `isValidLCDate`.

Reply:
Just to clarify you mean that the variable name here in the test should correspond to the explicit parameter name in isValidLCDate?